### ISSUE TITLE
amici 0.11.21 on Windows with MSVC

### ISSUE THREAD

**Issue by thomassligon** [AMICI-dev/AMICI#1641]
I just updated Python from 3.7 to 3.9, but installation of amici failed.
I am using MSVC 2022 Community with Python 3.9 and C++ 14.3
`C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Redist\MSVC\14.30.30704`
How does amici installation test the MSVC C++ version? I originally installed VS Desktop Development with C++ and, because of this error, also installed the standalone build tools. Am I missing something else?
```
  Changed cflags for amici with msvc to ['-IC:\\BLAS\\OpenBLAS-v0.3.10\\OpenBLAS-0.3.10', '/std:c++14']
  error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
  ----------------------------------------
  ERROR: Failed building wheel for amici
Failed to build amici
ERROR: Could not build wheels for amici, which is required to install pyproject.toml-based projects
----- Failed to install 'amici==0.11.21' -----

```

**Comment by dweindl** [comment]
Hi Tom, 

>   error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/

is not from AMICI directly, but (i guess) from `setuptools`. I don't have any Windows available for testing. Maybe there is more information on the internet. Generally, AMICI works fine with Python3.9 on Windows.

**Comment by stephanmg** [comment]
Could you please provide the error log file @thomassligon?

**Comment by dweindl** [comment]
P.S.: https://stackoverflow.com/a/55370133 - helps?

**Comment by thomassligon** [comment]
Hi Stephan, thanks for the response! What is the error log file, and where can I find it? I searched my disk for *.log and didn't find anything that looked right. So, I ran the install amici command and copied the Python output into a file, which I attached here.
[output.txt](https://github.com/AMICI-dev/AMICI/files/7940125/output.txt)

**Comment by thomassligon** [comment]
Thanks to both of you for the help. Daniel, your Stackoverflow suggestion is correct, except, of course, I often used Visual Studio Installer and also installed the C++ package. Otherwise, I wouldn't have been able to write what was in my original request. But, when I checked it today, this piece was missing, so I installed it, fixed a problem with my installation of swig, and now it is working. So, I'll close the issue.
